### ISSUE TITLE
Emotion Classification Accuracy (Multi-task fine-tuning for Generative model)

### ISSUE THREAD

**Issue by gentaiscool** [facebookresearch/EmpatheticDialogues#6]
I am curious about the emotion classification accuracy on the test set after multi-task fine-tuning training. Currently, I am running experiments on your dataset and would like to compare the results.

Looking forward to your reply. Appreciate your help and time.

Thank you

**Comment by EricMichaelSmith** [comment]
Thanks for reaching out! I just pinged the person who trained the emotion classifier to see if they can shed more light on the classification accuracy, and I'll get back to you when I find out about this.

**Comment by gentaiscool** [comment]
Thank you Eric

**Comment by EricMichaelSmith** [comment]
Hi! I heard back about the emotion classifier - unfortunately, we never explicitly measured the classification accuracy on the test set after multi-task finetuning.

**Comment by gentaiscool** [comment]
noted. thank you for your reply

**Comment by EricMichaelSmith** [comment]
Sure thing!
